src/apps/persistence/app_listing.clj

```diff
@@ -223,15 +224,21 @@
       (add-app-group-plus-public-apps-where-clause app-group-id username public-app-ids)
       (select))))
 
+(defn- add-date-limits-where-clause
+  [query {:keys [start_date end_date]}]
+  (if (and start_date end_date)
+    (where query {:end_date [between [(date->timestamp start_date) (date->timestamp end_date)]]}) query)
```

Comment:
I don't know if this is worth the bother, but it might be a bit confusing that if you don't provide both a start and end date, they'll be ignored. (May depend on what the UI/belphegor is doing with this, though). It should be equivalent to `BETWEEN` to add a `<=` and a `>=` separately, if you think it matters enough to change.